docs/software/apps-and-envs/lammps.md

```diff
@@ -99,7 +100,7 @@ n=$(( ntasks_per_node * numnodes ))
 mpirun -np $n lmp_gpu -input in.txt -sf gpu -pk gpu 2
 ```
 
-The following script illustrates how to run the LAMMPS binary built with the KOKKOS package `lmp_kokkos_cuda`
+The following script illustrates how to run the LAMMPS binary built with the KOKKOS package `lmp_kokkos_cuda` using 2 MPI processes on 2 CPU cores and 2 GPUs.
```

Comment:
- "using 2 MPI processes on 2 CPU cores and 2 GPUs." => "using 2 MPI processes on 2 CPU cores using 2 GPUs."?

Reply:
checked